docs/reference/status_helper.rst

```diff
@@ -54,3 +54,12 @@ You can now call your helper in your twig views using the following code:
 .. code-block:: twig
 
     {{ my_object|sonata_status_class(status_name, 'default_value') }}
+
+[Deprecated] Use the service with FlashManager
```

Comment:
Is it a good idea to add doc about something deprecated ?

Reply:
In most cases is it not. Here we remove some hacks, so will be nice to tell people how to use it in correct way, even if this is deprecated.

Reply:
if there is documentation about the right way to do it, I prefer to remove the deprecated doc.